conda/environment.yml

```diff
@@ -22,16 +22,17 @@ dependencies:
   - python=3.8
   - freecad=0.19.3
   - pivy=0.6.5 # later pivy versions (0.6.7 at the time of writing) crash our FreeCAD CAD viewer
-  - numpy=1.21.5
+  - numpy=1.21.6
```

Comment:
Should we be changing versions of deliberate dependencies in this PR?

Reply:
Oops that must have come in via something automatic, either way this should be changed to reflect 1.22.4 that went in the other day. Will fix, thanks